src/payment.gql

```diff
@@ -0,0 +1,139 @@
+extend schema
+  @link(url: "https://specs.apollo.dev/federation/v2.3", import: ["@composeDirective", "@extends", "@external", "@inaccessible", "@interfaceObject", "@key", "@override", "@provides", "@requires", "@shareable", "@tag"])
+
+"""A payment of an invoice or return"""
+type Payment
+  @key(fields: "id")
+{
+  """The uuid identifier of the product item"""
+  id: UUID!
+
+  """Payment Amount in EUR"""
+  amount: Float!
```

Comment:
all our other services use int and cents to avoid floating point weirdness, I would prefer if we keep it consistent across all services